plan.go

```diff
@@ -1,28 +1,34 @@
-package main
+package plan
 
 import (
-	"fmt"
+	//	"fmt"
 	"github.com/hashicorp/terraform/terraform"
-	"os"
+	"strings"
 )
 
-func main() {
-	file, err := os.Open("enablekms.plan")
-	if err != nil {
-		panic(err)
-	}
-
-	plan, err := terraform.ReadPlan(file)
-	if err != nil {
-		panic(err)
-	}
+const (
```

Comment:
These number should come from the terraform project and not reused by us

So just use terraform.DiffXYZ

https://github.com/hashicorp/terraform/blob/master/terraform/diff.go#L19

Reply:
This is not fixed ^^